test/Google.Api.Gax.Tests/ServiceSettingsBaseTest.cs

```diff
@@ -15,7 +15,13 @@ public class ServiceSettingsBaseTest
     {
         class TestSettings : ServiceSettingsBase
         {
-            public TestSettings Clone() => CloneInto(new TestSettings());
+            public TestSettings()
+            {
```

Comment:
Nit: Put open+close curly brackets on line above, as with the constructor below.